tests/test_viz.py

```diff
@@ -53,10 +53,9 @@ def test_dense_input_xor():
     Z = Z.reshape(xx.shape)
     ax.contourf(xx, yy, Z, alpha=.3, cmap=plt.cm.Paired)
     ax.axis('off')
-
     fig.savefig("test_model_xor.png")
-
-    dg = DenseGraph(model)
+    dg = DenseGraph(model, input_color='#FF0000', inner_color='#00FF00', output_color='#FFFF00',
```

Comment:
Please keep this test the same and add a new test just for your color map. If possible, the test should check that the output graph has the right colors